Application/Logic/CustomerLogic.cs

```diff
@@ -23,5 +41,10 @@ public CustomerLogic(ICustomerDao customerDao)
 
         return customer;
     }
+
+    private async Task<int> CustomerExistsAsync(Customer customer)
```

Comment:
This method is redundant, why just not call the CustomerDao method directly.

Reply:
done